### ISSUE TITLE
Running a11y does not work on windows

### ISSUE THREAD

**Issue by christian-fischer** [dynatrace-oss/barista#1038]
# Bug Report

## Expected Behavior

 I expect a11y-tests to work on windows.

## Current Behavior

Regardless of used browser (tested with several), the execution fails as soon as the browser tries to open the page at 0.0.0.0

## Possible Solution

As a workaround suggested by @lukasholzer replacing `"host": "0.0.0.0"` with `"host": "localhost"` seems to work.

## Steps to Reproduce

1. run `ng build demos` and 
2. `ng run demos:a11y-tests --browsers=firefox`
3. This leads to `Firefox can't establish a connection to the server at 0.0.0.0:49787`

## Context (Environment)

Windows

### Used Versions:

- **node**: 12,16,1
- **npm**: 6,13,4
- **@dynatrace/barista-components**: current master

## Detailed Description

I think the issue seems to be with trying to use 0.0.0.0 as a ip in the browser. As the bind address this is fine (all ips on the system); but as a target ip this is not really valid.

## Possible Implementation

Change angular.json:313 as outlined in the section "possible solution"

## Attachments
![image](https://user-images.githubusercontent.com/5974170/82217242-6de2c900-991a-11ea-97d6-14e878534275.png)

**Comment by lukasholzer** [comment]
Thank you very much for filing this detailed description! ✌️
Are you willing to contribute this fix for us as you work on a windows machine? From my perspective, there should not be a downside to switch the host to `localhost`.
